explain-indexes.md

```diff
@@ -0,0 +1,343 @@
++--------------------------------+---------+-----------+---------------+-----------------------------------+
+6 rows in set (0.00 sec)
+```
+
+The following statements will use the `IndexFullScan` operator to scan every row in the index:
+
+{{< copyable "sql" >}}
+
+```sql
+EXPLAIN SELECT SUM(intkey) FROM t1;
+EXPLAIN SELECT AVG(intkey) FROM t1;
+```
+
+```sql
++----------------------------+---------+-----------+--------------------------------+-------------------------------------+
+| id                         | estRows | task      | access object                  | operator info                       |
++----------------------------+---------+-----------+--------------------------------+-------------------------------------+
+| StreamAgg_20               | 1.00    | root      |                                | funcs:sum(Column#6)->Column#4       |
+| └─IndexReader_21           | 1.00    | root      |                                | index:StreamAgg_8                   |
+|   └─StreamAgg_8            | 1.00    | cop[tikv] |                                | funcs:sum(test.t1.intkey)->Column#6 |
+|     └─IndexFullScan_19     | 1010.00 | cop[tikv] | table:t1, index:intkey(intkey) | keep order:false                    |
++----------------------------+---------+-----------+--------------------------------+-------------------------------------+
+4 rows in set (0.00 sec)
+
++----------------------------+---------+-----------+--------------------------------+----------------------------------------------------------------------------+
+| id                         | estRows | task      | access object                  | operator info                                                              |
++----------------------------+---------+-----------+--------------------------------+----------------------------------------------------------------------------+
+| StreamAgg_20               | 1.00    | root      |                                | funcs:avg(Column#7, Column#8)->Column#4                                    |
+| └─IndexReader_21           | 1.00    | root      |                                | index:StreamAgg_8                                                          |
+|   └─StreamAgg_8            | 1.00    | cop[tikv] |                                | funcs:count(test.t1.intkey)->Column#7, funcs:sum(test.t1.intkey)->Column#8 |
+|     └─IndexFullScan_19     | 1010.00 | cop[tikv] | table:t1, index:intkey(intkey) | keep order:false                                                           |
++----------------------------+---------+-----------+--------------------------------+----------------------------------------------------------------------------+
+4 rows in set (0.00 sec)
+```
+
+This is considered more efficient than a `TableFullScan` because the width of the value in the index `(intkey + RowID)` is less than the width of the full row.
```

Comment:
```suggestion
In the above examples, `IndexFullScan` is more efficient than `TableFullScan` because the width of the value in the `(intkey + RowID)` index is less than the width of the full row.
```